lib/generators/active_record/templates/migration.rb

```diff
@@ -7,6 +7,8 @@ def up
       t.datetime   :invitation_accepted_at
       t.integer    :invitation_limit
       t.references :invited_by, :polymorphic => true
+      t.integer    :invitations_count, default: 0
+      t.index      :invitations_count
```

Comment:
Why do you need an index on `invitations_count`? Do you query users somewhere based only on invitation counts? And even if you do, the index doesn't seem selective enough.

Reply:
Yeah, I queried users based on that and displaying it on a view. I made it as a preventive matter if I recall correctly

Reply:
I doubt if the majority of people use it for that purpose (some kind of leaderboard), and it would lie as an unused index in their table. I personally think that this index does not belong to the gem migration but should be added separately (in their own migration) if someone needs that kind of functionality.

Reply:
You may be right, and probably is a better idea to have the index on a separate migration file, in case someone needs it

Reply:
Created a PR: https://github.com/scambra/devise_invitable/pull/830